### ISSUE TITLE
mjml warning of "beautify" option depreciation on watch, even with no config option set

### ISSUE THREAD

**Issue by rootwork** [mjmlio/mjml#2205]
**Describe the bug**

When I run `mjml -w index.mjml` I get this warning:

```
"beautify" option is deprecated in mjml-core and only available in mjml cli.
```

even though I haven't set any beautify option. This makes for a lot of noise in the watch console.

**To Reproduce**
1. `echo '<mjml><mj-body></mj-body></mjml>' > index.mjml`
2. `mjml -w index.mjml`
4. See warning

**Expected behavior**

`mjml -w` should watch and process a valid file without throwing warnings.

**MJML environment (please complete the following information):**
 - OS: Linux (Mint 20.1, Ubuntu 20)
 - MJML Version: 4.9.0
 - MJML tool: npm core module (mjml, not mjml-cli)

**Comment by iRyusa** [comment]
As beautify is set to true by default it will display this message until MJML5 you can just disable beautify for now

**Comment by rootwork** [comment]
My point is that MJML4 shouldn't output warnings when the user hasn't set any config options. MJML4 sets beautify by default, great, good for it, but if it's going to do that then it shouldn't _also_ warn the user about a setting it silently turned on.

Look at this from a new user's perspective: I run MJML for the first time on my file, and, crap, there's a warning! Beautify? But I didn't tell it to beautify anything. Why is it warning me about its own default behavior?

Either MJML4 should:

- not turn on beautify by default anymore; I assume this is no-go since it would change expected behavior and (maybe?) build scripts
- know to not display the depreciation warning unless the beautify config option is actually passed to it _by the user_

**Comment by TrySound** [comment]
Weird, here minify and beautify are omitted
https://github.com/mjmlio/mjml/blob/master/packages/mjml-cli/src/client.js#L213
And here defaults are set to false
https://github.com/mjmlio/mjml/blob/master/packages/mjml-core/src/index.js#L92-L103

Did I miss something?

**Comment by iRyusa** [comment]
Probbly this
https://github.com/mjmlio/mjml/blob/master/packages/mjml-cli/src/helpers/defaultOptions.js

**Comment by rootwork** [comment]
Note this only happens on MJML _watch_, which makes it even more surprising. Running the "steps to reproduce" I included in the OP with `mjml -r index.mjml -o index.html` results in no depreciation warning.

**Comment by kmcb777** [comment]
@rootwork We just released v4.9.1, this should be fixed you can try it out

**Comment by rootwork** [comment]
Yep, it works. Thank you!
